packages/grid/x-data-grid-generator/src/renderer/renderEditStatus.tsx

```diff
@@ -44,7 +44,7 @@ function EditStatus(props: GridRenderEditCellParams<string>) {
         onClose: handleClose,
       }}
       sx={{
-        height: 1,
+        height: '100%',
```

Comment:
I believe this is kind of confusing for people that don't know https://mui.com/system/sizing/#supported-values

Reply:
haha, right, well, happy to hear other's perspectives on this. I think that the main value is the feature is so that developers can do:

```suggestion
        height: 1/2,
```

we had to support height: 1 for consistency. `height: 2` generate a height of 2px 🙃. In MUI Core, we never use this API.

cc @mnajdova should we consider removing the documentation of it in https://mui.com/system/sizing/#supported-values?

Reply:
> cc @mnajdova should we consider removing the documentation of it in https://mui.com/system/sizing/#supported-values?

Agree, I remember another discussion some time ago about the same topic. Linking the comment to the v6 milestone for removal and yep, will remove the docs for it.

Reply:
Idea moved to https://github.com/mui/material-ui/issues/38583.